Gemfile

```diff
@@ -1,11 +1,11 @@
 source 'https://rubygems.org'
 
-platforms :windows do
-  gem 'tzinfo-data'
-end
```

Comment:
please keep this

Reply:
@johnnyshields it's invalid and causing other builds to fail. I'm not sure what value should be used since Bundler reports many but didn't see a generic windows related one.

Reply:
Please upgrade bundler. It's not invalid.

Reply:
No, sorry, I wasn't clear. That's what was being reported in GitHub Actions. I will try again once I resolve some other issues.